presto-product-tests/src/main/java/io/prestosql/tests/cli/PrestoCliLauncher.java

```diff
@@ -78,9 +78,18 @@ protected void launchPrestoCli(List<String> arguments)
 
     protected ProcessBuilder getProcessBuilder(List<String> arguments)
     {
-        return new ProcessBuilder(ImmutableList.<String>builder()
+        List<String> command = ImmutableList.<String>builder()
                 .add(JAVA_BIN, "-Xmx50m", "-cp", CLASSPATH, Presto.class.getCanonicalName())
                 .addAll(arguments)
-                .build());
+                .build();
+
+        String commandText = Joiner.on(' ').join(command);
+
+        return new ProcessBuilder(ImmutableList.of("bash", "-c", commandText));
+    }
+
+    static String escapeForBash(String value)
+    {
+        return '"' + value + '"';
```

Comment:
This is not correct.

```
requireNonNull(value, "value is null");
return "'" + value.replace("'", "'\\''") + "'";
```

Reply:
Anyway, i'd rather see the code without need for escape, and bash. Too error prone.